pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/ReadableSequentialDataTest.java

```diff
@@ -11,6 +20,38 @@ protected ReadableSequentialData emptySequence() {
         return new StubbedSequence(new byte[0]);
     }
 
+    @NonNull
+    private ReadableSequentialData throwingSequence() {
```

Comment:
This would test `StubbedSequence` only. Instead, it would be great to have this method as abstract in `ReadableSequentialTestBase` and implement it wherever needed. It would test all ReadableSequentialData implementations.

Reply:
Yes, that was my original idea, too. But I found it unfeasible to support this test in one of the descendants of the TestBase - I recall I just couldn't reach the particular branch that would throw the exception from the test that uses this sequence. That's the reason why it's sitting here.